solutions/swb-app/src/errors/errors.test.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+import { DatabaseError, isDatabaseError } from './databaseError';
+import { isNoKeyExistsError, NoKeyExistsError } from './noKeyExistsError';
+import { isNonUniqueKeyError, NonUniqueKeyError } from './nonUniqueKeyError';
+
+describe('custom error tests', () => {
+  let error: Error;
+
+  beforeAll(() => {
```

Comment:
Always use `beforeEach/afterEach` to start and only move to `beforeAll/afterAll` if you have setup that is very expensive.

Reply:
I would group all of the `is not *Error` tests together under a separate `describe` so that the `error` is only a part of that context.